R/z_animintHelpers.R

```diff
@@ -900,11 +900,11 @@ varied.chunk <- function(dt.or.list, cols){
     dt <- dt.or.list[, cols, with=FALSE, drop=FALSE]
     u.dt <- unique(dt)
     group.counts <- u.dt[, .N, by = group]
-    if(all(group.counts$N == 1)){
-      setDF(u.dt)
+    setDF(if(all(group.counts$N == 1)){
```

Comment:
hey @Faye-yufan did you know that the if statement returns whatever is the last expression in the if or else block? so we don't need to repeat the setDF inside the if/else, we can put it outside.

Reply:
That's new to me, thanks for letting me know!